src/main/java/core/basesyntax/model/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public interface AreaCalculator {
+    double getArea();
```

Comment:
I would say it is better here to use `calculateArea` method name as we actually do some computation rather than get stored value.